packages/collections/graphql/resolvers/User.js

```diff
@@ -13,6 +13,22 @@ const canAccess = (user, context) => {
   )
 }
 
+const progressFinishedThreshold = 0.85
```

Comment:
Where does this magic number come from?

We will probably have to make this configurable over the API at some point, no?

Reply:
I don't know if will ever use this from the API - but it kind of should be consistent with the frontend, where we only show "finished" or "Gelesen" when progress is 1 (100%). So that's a bit inconsistent in my mind.